source/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "node-fetch";
+declare module "parse-diff";
+declare module "lodash.includes";
+declare module "lodash.find";
+declare module "jest-runtime";
+declare module "jest-environment-node";
+
+declare module "*/package.json";
```

Comment:
it is ambient declaration for module does not have any type definition. By default TS compiler without allow impicit any does not accept to import plain CJS module without corresponding type definition (either `@types` or package-provided typings). This ambient declaration makes all those CJS modules `any` type, can be imported where it doesn't have type definition still.